storage/ndb/rest-server/data-access-rondb/src/rdrs_rondb_connection.hpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2022 Hopsworks AB
```

Comment:
2023

Reply:
Actually 2023, 2023 (first copyright year and year of update)

Reply:
fixed